### PR TITLE
[NUI] Add a Window constructor using WindowData

### PR DESCRIPTION
**Pull request by ANZ1217**
### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/Window/Window.cs [hunk 1]
```diff
@@ -154,6 +154,28 @@ public Window(string name, IBorderInterface borderInterface, Rectangle windowPos
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Creates a new Window with a specific name.<br />
```
Comment (hinohie) [inline src/Tizen.NUI/src/public/Window/Window.cs:158]: Summary don't have any information about windowData.

File: src/Tizen.NUI/src/internal/Interop/Interop.Window.cs [hunk 2]
```diff
@@ -35,6 +35,9 @@ internal static partial class Window
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_new_Window__SWIG_0")]
             public static extern global::System.IntPtr NewWindow();
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_New__WithWindowData")]
+            public static extern global::System.IntPtr New(string name, string className, global::System.Runtime.InteropServices.HandleRef windowData);
```
Comment (hinohie) [inline src/Tizen.NUI/src/internal/Interop/Interop.Window.cs:39]: Could you please move this function after line 33? I think it is familiar with New(~~). not NewWindow()

File: src/Tizen.NUI/src/public/Window/Window.cs (hunk 1 above)
Comment (ANZ1217) [reply src/Tizen.NUI/src/public/Window/Window.cs]: Done.

File: src/Tizen.NUI/src/internal/Interop/Interop.Window.cs (hunk 2 above)
Comment (ANZ1217) [reply src/Tizen.NUI/src/internal/Interop/Interop.Window.cs:36]: Done.

**Review by hinohie — APPROVED** [review]
Approved. Please merge this PR as "Squash and Merge", instead of "Rebase and Merge"

**Pull request merged**